k8s/aat/cluster-00/static-overlay/static/kube-slack/kube-slack.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: kube-slack
+  namespace: admin
+spec:
+  values:
+    envVars:
+      SLACK_USERNAME: aat-00-aks
```

Comment:
lol, kube-slack and fluxcloud use the same environment variables, but they're in a different order, it pains me 👅